ezidapp/management/commands/proc-download.py

```diff
@@ -235,7 +235,7 @@ def _prepareMetadata(
         return d
 
     def _writeAnvl(
-        self, f: typing.TextIO, id_model: ezidapp.models.identifier.SearchIdentifier, metadata: dict
+        self, f: typing.TextIO, id_model: ezidapp.models.identifier.Identifier, metadata: dict
```

Comment:
@sfisher Should this be `ezidapp.models.identifier.SearchIdentifier`?

Reply:
Thanks for catching this, I must've missed it when I updated things back.